src/linux/ebpf.hpp

```diff
@@ -61,6 +61,11 @@ namespace cgroups2 {
 // device access dynamically.
 Try<Nothing> attach(int fd, const std::string& cgroup);
 
+
+// Get the program ids of all programs that have been attached to a cgroup.
+// @param    cgroup   Absolute path of a cgroup.
+Try<std::vector<uint32_t>> attached(const std::string& cgroup);
```

Comment:
there appear to be several types of cgroup related bpf programs, but this is device specific, it would be good to put device specific stuff in a device:: namespace here, don't need to do it in this patch though